contracts/libs/FixedPointInt256.sol

```diff
@@ -25,15 +25,6 @@ library FixedPointInt256 {
         int256 value;
     }
 
-    /**
-     * @notice constructs an `FixedPointInt` from an unscaled int, e.g., `b=5` gets stored internally as `5**27`.
-     * @param a int to convert into a FixedPoint.
-     * @return the converted FixedPoint.
-     */
-    function fromUnscaledInt(int256 a) internal pure returns (FixedPointInt memory) {
-        return FixedPointInt(a.mul(SCALING_FACTOR));
-    }
```

Comment:
Can we keep it ? in case we need it somewhere in the future, then no need to update the library.

Reply:
I'm good with it, but then we should close the issue, 
I just randomly felt like working on some easy issues 😂